### PR TITLE
enable protobuf linter on CI

### LINKED ISSUE #318: Add protobuf linter to travis build
Follow up of #302

Add the protobuf linter to the CI. Fail the build if linter is reporting errors.

~Blocked by: #312~

### PR DESCRIPTION
**Pull request by husio**
enable protobuf linter on CI
fix #318

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,15 +55,12 @@ deps: tools
...
-lint: prototool-bin
-	prototool lint
-
-prototool-bin:
+lint:
 ifndef $(shell command -v prototool help > /dev/null)
-	@@curl -sSL https://github.com/uber/prototool/releases/download/v1.3.0/prototool-$(shell uname -s)-$(shell uname -m) -o $(HOME)/.bin/prototool
-	@@chmod +x $(HOME)/.bin/prototool
+	@go get github.com/uber/prototool/cmd/prototool
+	@go install github.com/uber/prototool/cmd/prototool
```
Comment (ethanfrey) [inline Makefile]: Doesn't go get also install the item if pointed to a main package and there is no `-d` flag?

I mean, it is nice how is, clearer and "safer", mainly just checking my understanding of the tools (been some months)

Comment (alpe) [reply Makefile]: @ethanfrey is right. The install line should be obsolete

> Get downloads the packages named by the import paths, along with their dependencies. It then installs the named packages, like 'go install'.

https://golang.org/cmd/go/#hdr-Download_and_install_packages_and_dependencies

File: .travis.yml [hunk 2]
```diff
@@ -34,6 +34,7 @@ install:
   rm -rf ${GOPATH}/src/github.com/tendermint/tendermint
 
 script:
+- make lint
```
Comment (alpe) [inline .travis.yml]: 👍

**Comment by husio** [comment]
I have removed `go install` and checked manually. It works! :tada:

**Pull request merged**